client/src/components/IndexPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+const IndexPage = props => {
+    const [trail, setTrail] = useState([])
```

Comment:
I would make this plural `trails` and `setTrails` since there should be many trails stored in this array